paparazzi/src/main/java/app/cash/paparazzi/Paparazzi.kt

```diff
@@ -284,6 +292,77 @@ class Paparazzi(
         }
   }
 
+  private fun initializeAppCompatIfPresent() {
+    lateinit var appCompatDelegateClass: Class<*>
+    try {
+      // See androidx.appcompat.widget.AppCompatDrawableManager#preload()
+      val appCompatDrawableManagerClass =
+        Class.forName("androidx.appcompat.widget.AppCompatDrawableManager")
```

Comment:
Should it also check for support library too for projects that have yet to jetify / migrate to androidx?